javascriptv3/example_code/personalize/src/personalize_createDatasetExportJob.js

```diff
@@ -0,0 +1,56 @@
+/* Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+SPDX-License-Identifier: Apache-2.0
+ABOUT THIS NODE.JS EXAMPLE: This example works with the AWS SDK for JavaScript version 3 (v3),
+which is available at https://github.com/aws/aws-sdk-js-v3.
+
+Purpose:
+createDatasetExportJob.js demonstrates how to create a dataset export job to export data from
+Amazon Personalize to an Amazon S3 bucket.
+See https://docs.aws.amazon.com/personalize/latest/dg/API_CreateDatasetExportJob.html.
```

Comment:
For more information, see https://docs.aws.amazon.com/personalize/latest/dg/API_CreateDatasetExportJob.html.